test/rekt/features/trigger/trigger_sink_resolution.go

```diff
@@ -132,6 +132,65 @@ func SourceToTriggerSinkWithDLSDontUseBrokers(triggerName, brokerName, brokerSin
 	return f
 }
 
+// source ---> broker +--[trigger<via1>]--> bad uri
+//                |   |
+//                |   +--[trigger<vai2>]--> sink2
+//                |
+//                +--[DLQ]--> sink1
+//
+func BadTriggerDoesNotAffectOkTrigger() *feature.Feature {
+	f := feature.NewFeature()
+
+	prober := eventshub.NewProber()
+	brokerName := feature.MakeRandomK8sName("broker")
+	via1 := feature.MakeRandomK8sName("via")
+	via2 := feature.MakeRandomK8sName("via")
+	dlq := feature.MakeRandomK8sName("dlq")
+	sink2 := feature.MakeRandomK8sName("sink2")
+
+	lib := feature.MakeRandomK8sName("lib")
+	f.Setup("install events", eventlibrary.Install(lib))
+	f.Setup("event cache is ready", eventlibrary.IsReady(lib))
+	f.Setup("use events cache", prober.SenderEventsFromSVC(lib, "events/three.ce"))
+	if err := prober.ExpectYAMLEvents(eventlibrary.PathFor("events/three.ce")); err != nil {
+		panic(fmt.Errorf("can not find event files: %s", err))
+	}
+
+	// Setup Probes
+	f.Setup("install sink1", prober.ReceiverInstall(dlq))
+	f.Setup("install sink2", prober.ReceiverInstall(sink2))
+
+	// Setup data plane
+	brokerConfig := append(broker.WithEnvConfig(), delivery.WithDeadLetterSink(prober.AsKReference(dlq), ""))
+	f.Setup("install broker", broker.Install(brokerName, brokerConfig...))
+	// Block till broker is ready
+	f.Setup("Broker is ready", broker.IsReady(brokerName))
+	prober.SetTargetResource(broker.GVR(), brokerName)
+
+	f.Setup("install trigger via1", trigger.Install(via1, brokerName, trigger.WithSubscriber(nil, "bad://uri")))
+	f.Setup("install trigger via2", trigger.Install(via2, brokerName, trigger.WithSubscriber(prober.AsKReference(sink2), "")))
+
+	// Resources ready.
+	f.Setup("trigger1 goes ready", trigger.IsReady(via1))
+	f.Setup("trigger2 goes ready", trigger.IsReady(via2))
+
+	// Install events after data plane is ready.
+	f.Setup("install source", prober.SenderInstall("source"))
```

Comment:
To install it after we need to use a different test phase
```suggestion
	// Install events after data plane is ready.
	f.Requirement("install source", prober.SenderInstall("source"))
```